cortex-snippets-client-ruby.gemspec

```diff
@@ -18,9 +18,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'cortex-client', '~> 0.10'
-  spec.add_dependency 'connection_pool', '~> 2.2'
-  spec.add_dependency 'addressable', '~> 2.5'
+  spec.add_dependency 'cortex-client', '~> 0.8.2'
```

Comment:
Both `cortex-client` and `addressable` have had their versions dropped here - can we bump these back up, and also make sure we're trusting them to be semantically versioned (AKA specifying `major.minor` rather than `major.minor.patch`)?